File: app/src/main/java/com/musicplayer/db/entity/Playlist.java
package com.musicplayer.db.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "playlists")
public class Playlist {
    @PrimaryKey(autoGenerate = true)
    public int id;
    
    public String name;
    public long createdAt;

    public Playlist(String name, long createdAt) {
        this.name = name;
        this.createdAt = createdAt;
    }
}

File: app/src/main/java/com/musicplayer/db/entity/Playlist.kt
package com.musicplayer.db.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlin.jvm.JvmField

@Entity(tableName = "playlists")
data class Playlist(
    @PrimaryKey(autoGenerate = true)
    @JvmField var id: Int = 0,
    @JvmField var name: String? = null,
    @JvmField var createdAt: Long = 0
)
